### DISCUSSION TITLE
Need some margin gap between caption and image in the viewer

### DISCUSSION THREAD

**Discussion by Ashritha-12** [igordanchenko/yet-another-react-lightbox#288] [Q&A] [answered]
I am unable to add margin between caption and image in the viewer. 

Tried , with lightbox variables but dont think it supports. 

Reason: I need this because when the image name is long  it just overlaps with the image and doesn't look good.

Can someone help here ?

**Comment by igordanchenko** [comment, marked as answer]
I'm afraid that's not supported in the current version. The captions are implemented as overlays and are legible even when positioned right over an image.

<img width="790" alt="Screenshot 2024-06-23 at 3 49 54 PM" src="https://github.com/igordanchenko/yet-another-react-lightbox/assets/64441155/b292a17f-1fa6-4c8c-b81d-674700d257aa">

You can increase `carousel.padding` value to avoid overlap, but that would add padding on all four sides of an image - not a margin that you are asking for.
